sympy/parsing/c/c_parser.py

```diff
@@ -494,10 +521,18 @@ def transform_parm_decl(self, node):
 
                 # If there is a child, it is the default value of the parameter.
                 lit = self.transform(child)
-                if (node.type.kind == cin.TypeKind.INT):
+                if node.type.kind in self._data_types["int"]:
+                    type = self._data_types["int"][node.type.kind]
```

Comment:
and type from here as they are already defined above

Reply:
That seems precise!